apps/TutorialServo/TutorialServo.h

```diff
@@ -0,0 +1,20 @@
+#ifndef TUTORIAL_SERVO_H
+#define TUTORIAL_SERVO_H
+
+#include "mbed.h"
+
+
+class TutorialServo
+{
+  public:
+  TutorialServo(PinName servoPin, float servoFrequency);
```

Comment:
an important part of object oriented programming is encapsulation. you have all your functions as public. This means users of this lib can access all the functions. It would be better if users can only access functions they should be accessing

Reply:
Okay, that makes sense. Which functions need to be made private?

Reply:
Well really that's up to you. How do you intend users to use this lib? I know this is just training and nobody will use this, but try and think like you're making this lib for someone else to use.

Reply:
I'm not entirely sure how private & public functions work in this context, is it cool if I ask you some questions on Discord on Thursday?

Reply:
sounds ood.